doc/src/input/mc.md

```diff
@@ -11,64 +30,121 @@ temperature = "500 K"
+The easiest way to handle frequencies is to use relative values. We will explain 
+this below in the given examples.
+
+Some moves can be specified to act on a single molecule or particle type. 
+These moves accept a `molecule` key whose value is a path to a configuration file 
+that can be read by [chemfiles][chemfiles]. 
+
+You can add the same move multiple times.
+For example, you can assign different amplitudes for different species in a 
+mixture to make sampling more efficient.
+You can also use the `molecule` type to freeze a species by assigning a move 
+for all but the frozen species.
+
+If you specify a molecule, it will be selected with the following algorithm:
 
 - Read the first frame of the file;
 - If the file does not contain any bonding information, try to guess the bonds;
 - Use the first molecule of the frame.
 
 [chemfiles]: chemfiles.github.io
 
-The same move type can be present more than once in the input, to have different
-amplitudes for different compounds for example.
+### Example
 
 ```toml
+# Simulation of a protein in water.
 [simulations.propagator]
 type = "MonteCarlo"
-temperature = "500 K"
+temperature = "300 K"
 moves = [
-    {type = "Translate", delta = "2 A", molecule = "H2O.xyz"},
-    {type = "Translate", delta = "0.5 A", molecule = "protein.pdb"},
+    # we have much more water in the system so we want to move it more often
+    # hence we set the `frequency = 100`
+    {type = "Translate", delta = "2 A", molecule = "H2O.xyz", frequency = 100},
+    # the single protein will be displaced only a small distance `delta = "0.05 A"`
+    {type = "Translate", delta = "0.05 A", molecule = "protein.pdb", frequency = 1},
 ]
 ```
 
-
 ### Translation
 
-The `Translate` move type will do small translations of a single molecule at the
-time. If the input contains the `molecule` key, the move will only apply to one
-molecule. If not, the move will apply to all molecules in the system. The
-`delta` key gives the amplitude of the translations, in distance unit.
+The `Translate` move changes the position of a single, randomly selected  molecule 
+by adding a random displacement vector to its center of mass.
+
+- Needed keys:
+    * `type = "Translate"`
+    * `delta` (string): Maximum amplitude for displacement.
+- Optional keys:
+    * `frequency` (float): Move frequency.  
+    * `molecule` (string): Select only the specified molecule type. The string 
+contains the path to the configuration file of the molecule.
+
+If the `molecule` key is used, the move will only apply to one
+molecule type. If not, the move will apply to all molecule types in the system. 
+The `delta` key is the maximum magnitude of the translation vector. The conjugated string 
+contains the value with unit of distance.
+
+#### Example
 
 ```toml
 [simulations.propagator]
 type = "MonteCarlo"
 temperature = "500 K"
 moves = [
     {type = "Translate", delta = "1 A", frequency = 2},
+    # for He, use a larger maximum displacement.
```

Comment:
Here, "He" would be translated by both the main move and the second move. Maybe specify/explicit it somewhere?